docs/subnets/subnet-upgrade.md

```diff
@@ -0,0 +1,155 @@
+
+That's why having a node upgrade strategy is absolutely vital, and you should always update to the latest AvalancheGo client immediately when it is made available.
+
+For a general guide on upgrading AvalancheGo check out [this tutorial](../nodes/maintain/upgrade-your-avalanchego-node.md). When upgrading Subnet nodes and keeping in mind the previous section, make sure to stagger node upgrades and start a new upgrade only once the previous node has successfully upgraded. Use the [Health API](../apis/avalanchego/apis/health.md#healthhealth) to check that `healthy` value in the response is `true` on the upgraded node, and on other Subnet validators check that [platform.getCurrentValidators()](../apis/avalanchego/apis/p-chain.md#platformgetcurrentvalidators) has `true` in `connected` attribute for the upgraded nodes `nodeID`. Once those two conditions are satisfied, node is confirmed to be online and validating the Subnet and you can start upgrading another node.
+
+Continue the upgrade cycle until all the Subnet nodes are upgraded.
+
+## Upgrading Subnet VM Plugin Binaries
+
+Besides the AvalancheGo client itself, new versions get released for the VM binaries that run the blockchains on the Subnet. On most Subnets, that is the [subnet-evm](https://github.com/ava-labs/subnet-evm), so this tutorial will go through the steps for updating the subnet-evm binary. The update process will be similar for updating any VM plugin bianry.
+
+All the considerations for doing staggered node upgrades as discussed in previous section are valid for VM upgrades as well.
+
+In the future, VM upgrades will be handled by the [AvalancheCLI tool](https://github.com/ava-labs/avalanche-cli), but for now we need to do it manually.
+
+Go to the [releases page](https://github.com/ava-labs/subnet-evm/releases) of the subnet-evm repository. Locate the latest version, and copy link that corresponds to the OS and architecture of the machine the node is running on (`darwin` = Mac, `amd64` = Intel/AMD processor, `arm64` = Arm processor). Log into the machine where the node is running and download the archive, using `wget` and the link to the archive, like this:
+
+```bash
+wget https://github.com/ava-labs/subnet-evm/releases/download/v0.2.9/subnet-evm_0.2.9_linux_amd64.tar.gz
+```
+
+This will download the archive to the machine. Unpack it like this (use the correct filename, of course):
+
+```bash
+tar xvf subnet-evm_0.2.9_linux_amd64.tar.gz
```

Comment:
Just for clarity—this tutorial should discuss subnet-evm v.0.2.8 and v0.2.9? We discuss v0.2.8 at the top and then start discussing v.0.2.9 here.

Reply:
0.2.9 is the current version so we use that, json upgrades are present since 0.2.8, which is why it was mentioned above.